### DISCUSSION TITLE
With dutch language, \textcite etc. raise issues for spelling error in 'Dummy'

### DISCUSSION THREAD

**Discussion by bertbaron** [valentjn/vscode-ltex#716] [Q&A]
When I run ltex with Dutch language (`"ltex.language": "nl"`) I get a warning like the following on every `\textcite`:

"'Dummy0': Er is een mogelijke spelfout gevonden. LTeX[MORFOLOGIK_RULE_NL_NL]"

A minimal working example:

```latex
\documentclass{article}
\usepackage[backend=biber]{biblatex}
\bibliography{bibliography}

\begin{document}
Zie \textcite{abc}. 
\end{document}
```

When I change to English no warning is shown.

Is this expected behavior? Or can I configure the dummy words somehow to become proper dutch words or do something else to fix or hide these warnigns?

**Comment by bertbaron** [comment]
In case anyone else ends up here before it is answered, my current workaround is to add the dummies explicitly to the dictionary, i.e. with:
```bash
for i in {0..99}; do echo "Dummy$i"; done >> .vscode/ltex.dictionary.nl.txt
```

**Reply by Sjorsa** [reply]
This doesn't work for me when there are words connected (without whitespace) to the `\textcite{}`, or in my case `\gls{}something`. This will give the error like `Dummy10something: Er is een mogelijke spelfout gevonden`. I can add all those cases to the ignore list too, but when I add a `\cite` or `\gls` earlier on in the document, all the next occurences move over, and I have to add them all over again.

I tried to add `Dummy*` to the ignore list, but wildcards don't seem to work.
